### ISSUE TITLE
Psych 3.0 on rubygems still shows ruby  >= 1.9.2

### ISSUE THREAD

**Issue by MaxLap** [ruby/psych#343]
If you go on Rubygems, right now, for psych 3.0.0, it still shows the required ruby as >= 1.9.2. (No idea why)

https://rubygems.org/gems/psych/versions/3.0.0

Probably because of this, bundler is installing psych 3.0.0 on my ruby 2.0.0 installation. Once this happens, i can't even use the `gem` command because it's trying to use psych 3.0.0 which raises a syntax error.

**Comment by hsbt** [comment]
It's my mistake. Pysch only supports `>= 2.2.2` versions.  see https://github.com/ruby/psych/commit/051440e93bd3cf1f2fc86602430d605fa608ff17

I will release 3.0.1 next week

**Comment by marcandre** [comment]
Maybe it would be a good idea to do a release ASAP, if only to fix this.

**Comment by marcandre** [comment]
I'm also wondering, will you have to yank 3.0.0?

**Comment by MaxLap** [comment]
FYI, for anyone landing on the issue as described in the first post, where even the `gem` command won't even work:

* You must `rm -rf` psych's gem directory. (Now the gem command should work)
* Then you can do `gem uninstall psych -v 3.0.0`. 
* Then, until version 3.0.0 gets fixed, you must add `gem 'psych', '< 3.0.0'` to your Gemfile. 
* You might have to run `bundle update` for it to accept your change of psych's version.

**Comment by stomar** [comment]
@hsbt please also consider #340 and related discussions for new gem releases.

**Comment by JoergWMittag** [comment]
Here's the error:

```
gems/psych-3.0.0/lib/psych/scalar_scanner.rb:146: syntax error, unexpected tIDENTIFIER, expecting ')' (SyntaxError)
      klass.new(yy, m, dd, hh, mm, ss+us/(1_000_000r), offset)
                                                    ^
```

[This comment added for Googleability of the fix. See also [*Why is Ruby throwing a syntax error whenever I use the gem command?*](https://stackoverflow.com/a/47623950/2988) on Stack Overflow.]

**Comment by marcandre** [comment]
@hsbt Unless the 3.0.1 supports Ruby 2.0, I don't see much other choice than to yank the 3.0.0 release. If so, please consider doing it now.

**Comment by hsbt** [comment]
I released psych-3.0.1 and yanked 3.0.0.

Note: Ruby 2.0.0 is already EOL status. You should update Ruby 2.3 or 2.4.
